### DISCUSSION TITLE
Import is very slow. Takes about 0.35 seconds which is too much.

### DISCUSSION THREAD

**Discussion by JosselinSomervilleRoberts** [sqlalchemy/sqlalchemy#11754] [Usage Questions]
### Describe the bug

Import is very slow. Takes about 0.35 seconds which is too much.

### SQLAlchemy Version in Use

2.0.30

### DBAPI (i.e. the database driver)

sqlite

### Database Vendor and Major Version

SQLite

### Python Version

3.11.9

### Operating system

CentOS Stream 9

### To Reproduce

```python
import time
t0 = time.time()
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy.orm import relationship
from sqlalchemy.orm.session import Session
from sqlalchemy.sql.schema import Column, ForeignKey
from sqlalchemy.types import JSON, Boolean, Float, Integer, String
t1 = time.time()
print(f"Imported SQLAlchemy time: {t1 - t0}")
```

### Error

Imported SQLAlchemy time: 0.3302574157714844

**Comment by CaselIT** [comment]
Hi

Not really sure how to fix. Do you have suggestions?

**Reply by JosselinSomervilleRoberts** [reply]
No, I just wanted to check that this was an expected behaviour as I have seen that old versions of sqlalchemy were faster to import.  See [link](https://github.com/sqlalchemy/sqlalchemy/issues/5681). What happened since then?
